### ISSUE TITLE
Hook up a backend

### ISSUE THREAD

**Issue by tommysullivan** [bob-bins/resume#5]
For persistant data storage we need some kind of backend. We can explore firebase.io, elasticsearch, mongodb, couchdb, and others.

I guess I am thinking of this "resume" site more as a template for building a full stack application so while your resume itself might not need a database of sorts, we can still hook one up to explore.

**Comment by bob-bins** [comment]
Yeah that's exactly what I was planning on. A personal website that includes a little aboutme section, my work history, and eventually a section for interactive web-projects such as physics models, games, sounds generator, etc. that I make for fun and learning.
